queuebot/cogs/queue/__init__.py

```diff
@@ -315,7 +336,7 @@ def wait_check(msg):
 
             vs_message = await queue.send(emote_sequence)
             for this_emoji in temp_emotes:
-                await vs_message.add_reaction(this_emoji)
+                await vs_message.add_reaction(NUMERIC_EMOTES[temp_emotes.index(this_emoji)])
```

Comment:
should add error handling for 6+ so we don't see a mysterious error just in case

Reply:
also you should be able to concat instead of a lookup table

Reply:
6+ Error handling is already handled in an earlier part of the code, and what do you mean by the second?

Reply:
alright then